### PR TITLE
fix: stories drawer to menu-button

### PR DESCRIPTION
**Pull request by eljefe223**
## New Behavior

Fixes stories so all arg controls work and better aligns with new storybook 8 API

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

File: packages/web-components/src/link/link.stories.ts [hunk 1]
```diff
@@ -1,71 +1,143 @@
...
+      description: 'Hints at the language of the referenced resource.',
+      name: 'hreflang',
+      table: { category: 'attributes', type: { summary: 'string' } },
+    },
+    ping: {
+      control: 'text',
+      description: 'The ping attribute.',
+      name: 'ping',
+      table: { category: 'attributes', type: { summary: 'string' } },
+    },
```
Comment (davatron5000) [inline packages/web-components/src/link/link.stories.ts:50]: I realize it's probably in the spec but I think it's disabled in Firefox and probably not recommended to use. According to https://caniuse.com/ping

> While still in the WHATWG specification, this feature was removed from the W3C HTML5 specification in 2010

No action necessary on this PR but may need a followup issue to remove.

**Review by davatron5000 — APPROVED** [review]
Looks good to me. I do go a bit back and forth on whether adding all the link and button attrs that emulate native html attributes adds clarity or noise in the storybook context. Most of those attrs aren't useful in a storybook story where you'd likely be looking for knobs to change/update the component. But from a documentation perspective, it's thorough so I think we start here.

File: packages/web-components/src/link/link.stories.ts (hunk 1 above)
Comment (eljefe223) [reply packages/web-components/src/link/link.stories.ts:50]: I'll remove

**Pull request merged**